content/tutorials/publish-subscribe.textile

```diff
@@ -737,12 +780,24 @@ blang[android].
   "See this step in Github":https://github.com/ably/tutorials/commit/pubsub-android-step4
 
 blang[javascript].
+  Add the call to @channel.publish@ to the last line of code within the @doPubSub()@ function:
+
   ```[javascript]
-    var channel = realtime.channels.get("sport");
-    channel.publish("update", { "team": "Man United" });
+    async function doPubSub() {
+      await realtime.connection.once("connected");
+      console.log("Connected to Ably!");
+
+      const channel = realtime.channels.get("sport");
+
+      await channel.subscribe((msg) => {
+        console.log("Received: " + JSON.stringify(msg.data));
+      });
+
+      await channel.publish("update", { team: "Man United" });
+    }
   ```
 
-  "See this step in Github":https://github.com/ably/tutorials/commit/pubsub-javascript-step4
+  Then, refresh your browser tab and watch the message payload being displayed in the browser console.
```

Comment:
We do ask them to load example.html in the section on downloading the tutorial from the GitHub, but I don't think we even ask them to load example.html into the browser in the main steps, so if they skipped the repo download and went straight into the main steps, they may get confused on what to load into the browser. Perhaps something along the lines of "If example.html is not loaded in the browser refresh the browser tab" or an earlier discrete step on loading example.html.

Reply:
Good point. Does the following work for you? [703effb](https://github.com/ably/docs/pull/1501/commits/703effb5d5f7fa6be0e30fd68ff86bc4d68786da)